config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -1306,7 +1306,7 @@ periodics:
         - name: CONTROL_PLANE_MACHINE_COUNT # CAPZ config
           value: "1"
         - name: KUBERNETES_VERSION # CAPZ config
-          value: "latest-1.28"
+          value: "latest"
```

Comment:
`latest` means k8s master branch?

Reply:
yes

Reply:
it may break sometimes, you are sure you want to use this master branch?

Reply:
Yes. I have verified locally and want to see if it works in testgrid.